packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/modal/ReactModalHostManager.java

```diff
@@ -158,6 +158,12 @@ public Map<String, Object> getExportedCustomDirectEventTypeConstants() {
     return eventTypeConstants;
   }
 
+  @Override
+  @ReactProp(name = "interactiveDismissal")
```

Comment:
`enableGesture` like RNSScreen seems like a better name?

Reply:
'interactiveDismissal' matches the terminology Apple uses for this feature, so I'd argue no, but happy to revise to match consensus.